scripts/common.py

```diff
@@ -67,7 +73,7 @@ def is_linux_os():
 
 def is_x64_architecture():
     """Check if the architecture is on X64."""
-    return ARCHITECTURE == 'x86_64'
+    return ARCHITECTURE.endswith('64')
```

Comment:
Perhaps explain more in the docstring at least two possible values that warrant this not being an equality check, or add a link to something authoritative that says this is the thing to do.

Reply:
Changed the implementation and added reference.